### PR TITLE
Visitor can navigate the site 

### PR DESCRIPTION
**Pull request by beccaburns**
## Navigation Bar Feature
[PT-Link](https://www.pivotaltracker.com/story/show/168829451)

## PR Content
In this PR, we include a navigation bar that is styled with Semantic UI CDN. We created a feature test with corresponding step definition. 

## Learning Points
We learned how to utilize Semantic UI within a ruby on rails application. We also learned the haml syntax and how to use resources for HTML/HAML conversion.

IMG_7393_(1)

Screen Shot 2019-10-02 at 4 13 44 PM

### CODE REVIEW

File: features/step_definitions/basic_steps.rb.old [hunk 1]
```diff
@@ -0,0 +1,7 @@
+When("I'm on the landing page") do
```
Comment (oliverochman) [inline features/step_definitions/basic_steps.rb.old]: whats up with this file?

File: features/nav_bar.feature [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Feature: Visitor can view navigation bar
+  As a visitor,
+  In order to navigate the site
+  I would like to use a navbar
+
+  Scenario: Viewing a nav bar with a log in bar
+    When I'm on the landing page
```
Comment (oliverochman) [inline features/nav_bar.feature]: `Given I'm on the landing page`

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Review by oliverochman — COMMENTED** [review]
Can you post a screenshot of how it looks

**Review by AlexSaxena — APPROVED** [review]

File: features/step_definitions/basic_steps.rb.old (hunk 1 above)
Comment (shsteindorsson) [reply features/step_definitions/basic_steps.rb.old]: It should be removed

**Review by tochman — CHANGES_REQUESTED** [review]
Please remove the CDN. Look at LESS and make sure you have the right js dependencies for SemanticUI installed.

**Review by oliverochman — APPROVED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Pull request merged**